server/source/controllers/todos.ts

```diff
@@ -93,6 +93,7 @@ const deleteTodo = async (req: Request, res: Response) => {
 
 
 const addTodo = async (req: Request, res: Response) => {
+  await delay(800);
```

Comment:
why? :)

Reply:
@skyboyer 
My backend responds too fast and there is almost no network delay. So I just wanted to get a little closer to the real environment with help of artificial delay. At least I can see that the loader fires every time.

Reply:
@skyboyer 
Do you think I should remove the delay from each router handler?

Reply:
yes, since you can throttle network speed right in Chrome DevTools -> Network if you need to check how app behaves on slow internet

Reply:
@skyboyer 
fixed.